rmgpy/tools/loader.py

```diff
@@ -39,19 +39,23 @@
 from rmgpy.solver.liquid import LiquidReactor
 from rmgpy.solver.base import TerminationConversion
 
-def loadRMGJob(inputFile, chemkinFile=None, speciesDict=None, generateImages=True, useJava=False, useChemkinNames=False):
+def loadRMGJob(inputFile, chemkinFile=None, speciesDict=None, generateImages=True, useJava=False,
```

Comment:
You can also update line 58 of rmgpy/solver/baseTest.py so that it doesn't check duplicates. I think you can do the same for line 81 of rmgpy/reduction/input.py but I don't know if non-RMG mechanisms can be used.